.github/workflows/ci.yaml

```diff
@@ -28,47 +28,44 @@ jobs:
       - name: Run pre-commit checks
         uses: pre-commit/action@v3.0.0
 
-  linux:
-    name: "Linux - unit tests - Python ${{ matrix.PYTHON_VERSION }}"
+  unit-tests:
+    name: "unit tests - Python ${{ matrix.PYTHON_VERSION }} - SA ${{ matrix.SA_VERSION }}"
     runs-on: ubuntu-latest
     env:
       CI: True
     strategy:
       fail-fast: true
       matrix:
-        PYTHON_VERSION: ['3.8', '3.9', '3.10']
+        # PYTHON_VERSION: ['3.8', '3.9', '3.10']
+        PYTHON_VERSION: ['3.10']
+        SA_VERSION: ['<=2.0', '>=2.0']
```

Comment:
there's now an `SA_VERSION` matrix element for most backends (where it makes sense) that also tests for the newer `sqlalchemy` version.
todo: we should pin this down, probably, since we can't guarantee every new version above 2.0